charts/vh-booking-queue-subscriber/values.dev.template.yaml

```diff
@@ -2,12 +2,14 @@
 java:
   image: '${IMAGE_NAME}'
   releaseNameOverride: ${RELEASE_NAME}
+  environment:
+    QUEUENAME: marco-dev-booking
```

Comment:
just a reminder to change these back before merging

Reply:
yes sure